worker/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "head-start-ttadp-worker",
+  "version": "1.0.0",
+  "description": "Job worker for the Office of Head Start Training & Technical Assistance Data Platform",
+  "main": "worker.js",
+  "scripts": {
+    "server": "nodemon  src/worker.js",
+    "lint": "eslint src",
+    "lint:ci": "eslint -f eslint-formatter-multiple src",
+    "test": "jest src",
+    "test:ci": "cross-env JEST_JUNIT_OUTPUT_DIR=reports JEST_JUNIT_OUTPUT_NAME=unit.xml POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub CURRENT_USER_ID=5 CI=true jest src --coverage --reporters=default --reporters=jest-junit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hhs/Head-Start-TTADP.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hhs/Head-Start-TTADP/issues"
+  },
+  "homepage": "https://github.com/hhs/Head-Start-TTADP#readme",
+  "dependencies": {
+    "aws-sdk": "^2.841.0",
+    "axios": "^0.21.1",
+    "bull": "^3.20.1",
+    "dotenv": "^8.2.0",
+    "form-data": "^3.0.0",
+    "pg": "^8.5.1",
+    "sequelize": "^5.22.3",
+    "throng": "^5.0.0",
+    "winston": "^3.3.3"
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.3",
+    "jest": "^26.6.3",
+    "jest-junit": "^12.0.0",
+    "nodemon": "^2.0.7"
+  }
```

Comment:
We need to add coverage checks like in frontend and backend. Please set all 4 checks to 90 so we don't need to increase later.